packages/storybook-renderer/src/render.ts

```diff
@@ -129,6 +129,11 @@ function addStory({
   resizeFn: ApplicationResizeFunction;
   storyObject: StoryFnPixiReturnType;
 }): EventHandler {
+
+  if(storyObject.startup) {
+    storyObject.startup(app);
+  }
+
```

Comment:
Ah, I was trying to avoid the story having to know about `app` at all, but I assume it must be necessary for something?

Reply:
Yeah in pixi-ui we need to override `app.renderer.events.rottBoundary.moveOnAll` property

I couldn't find a way to do that without passing the app

Reply:
I'm wondering if we can leverage storybook decorators and React hooks to do this instead. I did originally play with an API like that but changed my mind in favour of the current solution. If we did go that way it would look something like this (as per [storybook example](https://storybook.js.org/blog/how-to-build-connected-components-in-storybook/)):

```jsx
const AppContext = createContext();
export const useApp = () => useContext(AppContext);

const withApp = (Story, context) => {
  // We can pass an application instance via `context` although it's a little ugly
  const { app } = context.parameters;
  
  // AppProvider is an 
  return (
    <AppContext.Provider value={app}>
      <Story />
    </AppContext.Provider>
  );
}

export const decorators = [withApp];
```

A `Story` could then access `app` like so:

```
const SomeStory = () => {
  const app = useApp();
  
  return {
    view: etc
  }
}
```